### PR TITLE
Docs 3655 custom metrics dec17

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+.. Leave the *Collapsible* option with the default of *True*.
+.. Click *Finish*. +
+You can repeat these steps to add multiple key values. +
+This image shows example values for a custom metric that tracks t-shirt sales.
++
+image::metrics/custom-metrics-connector-config.png[]
++
+Select the *Throw runtime exception on validation failure* option if you want the flow to break if invalid values are entered for the metric. If this option is left unchecked, an error will be logged, but the flow won't break.
+
+== Deploy the application
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc]: Please use Title Case in headings

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+. Add the dimension for the metric:
+.. In *Metric name*, enter a meaningful value, for example "sales." 
+.. In *Enter contextual information* select *Edit inline* to enter the name and value for the dimension. 
+.. Click the green plus on the left to enter the values.
+.. In the *Dimension* dialog box, enter the *Name* and *Value* for the dimension.
+.. Leave the *Collapsible* option with the default of *True*.
+.. Click *Finish*. +
+You can repeat these steps to add multiple key values.
+. Enter the facts for the metric. Values for measurements are numerical.
+.. In *Enter measurements* select *Edit inline* to enter the name and value for the metric.
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc:71]: Need to discuss the other options -- (Expression, Edit Inline, and Bean Reference) -- Even if they're not part of your demo, users need to know what they are for.

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 3]
```diff
@@ -0,0 +1,96 @@
+= Anypoint Custom Metrics Connector
+
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc]: Should remove this blank line after the title. All other docs I've seen don't have blank lines after a title.

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using Custom Dashboards.
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc]: Please say what you mean by hybrid or at least provide a link to the Marketing hybrid site. Most people seeing hybrid think cars.

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+[NOTE] 
+====
+To use the Anypoint Custom Metrics connector, you must have a Titanium subscription. +
+This feature is currently supported only for hybrid applications.
+====
+
+== Prerequistes
+
+* Mule runtime engine 4.1.x
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc:18]: Marketing may use lowercase "runtime" but our docs use "Runtime"

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+.. Click *Finish*. +
+You can repeat these steps to add multiple key values.
+. Select the *Throw Runtime Exception on Validation Failure* option if you want the flow to break if invalid values are entered for the metric. If this option is left unchecked, an error will be logged, but the flow won't break.
+
+
+== Configure Custom Metrics Values Using Anypoint Studio
+
+. Search Anypoint Exchange for the Anypoint Custom Metrics connector. +
+The connector has one operation called "Send custom metric." 
+. Add the *Send Custom Metric* operation to your application flow. +
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc:57]: Exchange in Studio does not let one add an operation. Users must click "Add to project" to add the connector to their project. Please discuss this. The operation only is relevant after adding the connector to the project and you drag the operation to the canvas.

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+.. Click the green plus on the left to enter the values.
+.. In the *Dimension* dialog box, enter the *Name* and *Value* for the dimension.
+.. Leave the *Collapsible* option with the default of *True*.
+.. Click *Finish*. +
+You can repeat these steps to add multiple key values.
+. Enter the facts for the metric. Values for measurements are numerical.
+.. In *Enter measurements* select *Edit inline* to enter the name and value for the metric.
+.. In the *Fact* dialog box, enter a *Name* and numerical *Value* for the fact. +
+[NOTE]
+For facts, the value must always be numerical. This is not validated at design time, but it will cause a runtime exception once deployed.
```
Comment (george-hoffer) [inline modules/ROOT/pages/metrics/custom-metrics-connector.adoc:74]: Should say "For a Fact" instead of "facts"

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc (hunk 2 above)
Comment (valkyrie69) [reply modules/ROOT/pages/metrics/custom-metrics-connector.adoc:71]: I'll have to add it later.

File: modules/ROOT/pages/metrics/custom-metrics-connector.adoc (hunk 7 above)
Comment (valkyrie69) [reply modules/ROOT/pages/metrics/custom-metrics-connector.adoc:74]: There can be more than one fact.

**Pull request merged**